pil-splitter.c

```diff
@@ -53,6 +53,7 @@ int main(int argc, char **argv)
 	size_t phoff;
 	bool is_64bit;
 	size_t offset;
+	size_t hashoffset;
```

Comment:
You added this local variable and assign to it, but I don't see it being read anywhere?

Reply:
@MarijnS95 done